app/manager-frontend/src/pages/Agent/list.jsx

```diff
@@ -31,6 +31,13 @@ export const AgentsList = () => {
     await mutate();
     onClose();
     setIsLoading(false);
+    toast({
+      position: "top",
+      title: `Agent ${selectedAgent?.name} deleted!`,
```

Comment:
```suggestion
      title: `Agent ${selectedAgent.name} deleted!`,
```
⛏️ If the toast only pops up when an agent is selected, you shouldn't need to check if it's undefined.